docs/review_apps.md

```diff
@@ -6,14 +6,9 @@ Both services are accessible at the single URL. By default, the base path will r
 
 ## Deployment
 
-Pending a migration to AKS, the current process for provisioning a review app can be lengthy and somewhat unstable. The build and deploy action can fail for a number of reasons, as can the deployed app. This includes:
+In order to deploy a review app you can either use the make target or raise a pull request and label the deployment with 'deploy-aks' and this will trigger the GHA workflow to deploy the review app based on the branch name.
```

Comment:
Maybe mention it deploys container based postgres and redis and the data is NOT PERSISTENT